client/checkout/blocks/index.js

```diff
@@ -160,7 +160,7 @@ if ( getUPEConfig( 'isWooPayEnabled' ) ) {
 }
 
 if ( getUPEConfig( 'isPaymentRequestEnabled' ) ) {
-	if ( getUPEConfig( 'isTokenizedCartPrbEnabled' ) ) {
+	if ( getUPEConfig( 'isTokenizedCartEceEnabled' ) ) {
 		registerExpressPaymentMethod(
 			tokenizedCartPaymentRequestPaymentMethod( api )
```

Comment:
I decided not to rename this (and the `tokenized-payment-request` directory). I think it's ok to have that work to be separate 👍